projects/mac_ai/testing/podman.py

```
@@ -79,56 +79,67 @@ def prepare_from_gh_binary(base_work_dir):
 
 
 def _get_repo_podman_path(base_work_dir):
-    system = config.project.get_config(f"remote_host.system", print=False)
-    version = config.project.get_config(f"prepare.podman.repo.version", print=False)
+    system = config.project.get_config("remote_host.system", print=False)
```

Ruff (0.8.2)

Local variable `system` is assigned to but never used

Remove assignment to unused variable `system`

(F841)